crates/sui-core/src/authority.rs

```diff
@@ -1386,85 +1373,9 @@ impl AuthorityState {
 
     pub fn handle_checkpoint_request(
         &self,
-        request: &CheckpointRequest,
+        _request: &CheckpointRequest,
```

Comment:
I am actually not sure if this is going to be used? The new checkpoint sync probably going to have it's own request format?

Reply:
I was hoping to keep this API there so that we could still query each validator regarding their latest checkpoint cert.
Do you think that's still useful?

Reply:
I would like to have a mechanism to ask validators for their latest checkpoint and any cert of signature on it. This shortcuts the discussions around how soon we can have full nodes accessing the checkpoint data, ie whether we wait for a canonical cert of we can read a few sigs before, and trust the checkpoint (without a canonical cert yet).

Reply:
I am not immediately sure how this API is going to be used. Data dissemination I guess going to work through anemo
@gdanezis For the early exchange of signatures I think about just reusing narwhal payload verification.
If we decide to keep it perhaps lets just return an error for now, and not panic in the handler, GRPC method that panics validator seems like a BFT hazard